plugins/cpp/model/include/model/cppdoccomment.h

```diff
@@ -26,7 +26,7 @@ struct CppDocComment
   unsigned long long contentHash;
 
   #pragma db not_null
-  std::string contentHTML;
+  std::string content;
```

Comment:
I know in our current development branch the database model is moldable as it is, but why aren't we calling this `contentMD`?

Reply:
I don't know if it is worth including the format name in the attribute. We can do that if you wish. By the way a next improvement would be storing only the location of doc comments instead of its content.